lighthouse-plugin-publisher-ads/utils/resource-classification.js

```diff
@@ -203,7 +272,9 @@ function isAdScript(url) {
  * @return {boolean}
  */
 function isAdRequest(request) {
-  return isAdSenseAdRequest(request) || isGptAdRequest(request);
+  return isAdSenseAdRequest(request) ||
```

Comment:
note: broke into multiple lines because I was getting a pre-commit linting error about the line being too long.